README.md

```diff
@@ -408,6 +410,32 @@ Example:
       thing'
 ```
 
+### Action: `append`
+
+Appends a string on the end of a given file. File must already exist. If no
+newline at end of `with` parameter, one will be added unless
+`skip_ensure_newline` is set to `true`.
+
+If you need to remove an existing trailing newline before appending, use
+`regex_replace` instead.
+
+Params:
+- `paths`: List of files and/or directory trees to append to end of.
+  May use template expressions (e.g. `{{.my_input}}`).
```

Comment:
Let's add `Directories will be crawled recursively and every file underneath will be processed.` to the end of the docs for `paths`; we're adding that in #123 to every action doc.